.github/ISSUE_TEMPLATE/BUG-REPORT.yaml

```diff
@@ -3,18 +3,6 @@ description: File a bug report to help us improve
 title: "[Bug]: "
 labels: ["bug"]
 body:
-  - type: markdown
```

Comment:
Do we want to keep any of this?

Reply:
Everything remaining in 6.x is maintained by us.